index.ts

```diff
@@ -52,8 +55,16 @@ app.use(notFoundHandler)
 
 app.use(ErrorHandler)
 
+Product.associate()
+Review.associate({ Product, User })
```

Comment:
```suggestion

```
You can use  hasMany or hasOne, belongs to instead, and its better if it done on model instead in index.ts

Reply:
Thank it is corrected